qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/IndexingIT.java

```diff
@@ -120,6 +125,60 @@ public void testIndexing() throws IOException {
         }
     }
 
+    public void testAutoIdWithOpTypeCreate() throws IOException {
+        final String indexName = "auto_id_and_op_type_create_index";
+        StringBuilder b = new StringBuilder();
+        b.append("{\"create\": {\"_index\": \"").append(indexName).append("\"}}\n");
+        b.append("{\"f1\": \"v\"}\n");
+        Request bulk = new Request("POST", "/_bulk");
+        bulk.addParameter("refresh", "true");
+        bulk.setJsonEntity(b.toString());
+
+        switch (CLUSTER_TYPE) {
+            case OLD:
+                Request createTestIndex = new Request("PUT", "/" + indexName);
+                createTestIndex.setJsonEntity("{\"settings\": {\"index.number_of_replicas\": 0}}");
+                client().performRequest(createTestIndex);
+                break;
+            case MIXED:
+                Request waitForGreen = new Request("GET", "/_cluster/health");
+                waitForGreen.addParameter("wait_for_nodes", "3");
+                client().performRequest(waitForGreen);
+
+                Version minNodeVersion = null;
+                Map<?, ?> response = entityAsMap(client().performRequest(new Request("GET", "_nodes/plugins")));
```

Comment:
Unsure if `plugins` is a copy-paste error or if it has some benefit here?

Reply:
copy paste error.